cmd/ef_tests/levm/report.rs

```diff
@@ -2,52 +2,102 @@
 // The cost to add this would be to return a Result<(), InternalError> in EFTestsReport methods
 
 use colored::Colorize;
-use std::fmt;
+use std::{collections::HashMap, fmt};
 
 #[derive(Debug, Default)]
 pub struct EFTestsReport {
+    group_passed: u64,
+    group_failed: u64,
+    group_run: u64,
+    test_reports: HashMap<String, EFTestReport>,
+    passed_tests: Vec<String>,
+    failed_tests: Vec<(String, (usize, usize, usize), String)>,
+}
+
+#[derive(Debug, Default, Clone)]
+pub struct EFTestReport {
     passed: u64,
     failed: u64,
-    total: u64,
-    passed_tests: Vec<String>,
-    failed_tests: Vec<(String, String)>,
+    run: u64,
+    // passed_tests: Vec<String>,
```

Comment:
Try not to propose commented code when merging to `main`.
```suggestion
```

Reply:
Done both in [Remove commented code](https://github.com/lambdaclass/lambda_ethereum_rust/pull/1175/commits/5c7d971ca53da3183a80427500dea84522f0baeb)